Vostok.Metrics.System/Host/HostNetworkUtilizationCollector.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Vostok.Metrics.System.Helpers;
+
+namespace Vostok.Metrics.System.Host
+{
+    internal class HostNetworkUtilizationCollector
+    {
+        private readonly DerivativeCollector receivedBytesPerSecondCollector;
+        private readonly DerivativeCollector sentBytesPerSecondCollector;
+
+        public HostNetworkUtilizationCollector()
+        {
+            receivedBytesPerSecondCollector = new DerivativeCollector();
+            sentBytesPerSecondCollector = new DerivativeCollector();
+        }
+
+        public void Collect(HostMetrics metrics, long newReceivedBytesValue, long newSentBytesValue)
+        {
+            metrics.NetworkReceivedBytesPerSecond = receivedBytesPerSecondCollector.Collect(newReceivedBytesValue);
```

Comment:
We should probably round these to integer values. Stuff like `2657,21185533305` bytes per second isn't terribly useful :)